### ISSUE TITLE
Error: It looks like you are using Playwright Sync API inside the asyncio loop. Please use the Async API instead.

### ISSUE THREAD

**Issue by JasonCrowe** [daijro/camoufox#82]
### Describe the bug:

This is a tricky bug for me. It only happens on PC, not on Mac. 
I'm not using any async code. 
I'm not using it in a notebook.
I am using proxies.

### Screenshots:
None

### To Reproduce:
The code loops through records from a database, opens camoufox.
`with Camoufox(addons=[addon_path], proxy={'server': f'http://{proxy}', 'username': '', 'password': ''}, geoip=True) as browser:`
 grabs data, then starts the loop again.

It will run many times, and then I'll get the following error:
**"Error: It looks like you are using Playwright Sync API inside the asyncio loop. Please use the Async API instead."**
and every loop after gives the same error until I restart the program.

### Version:
Pip package:    v0.3.8
Camoufox:       v130.0.1-beta.13 (Latest supported: v132.0-beta.15)

Windows and Mac should be on the same version. Above is the Mac, I am working on getting the Windows version.

**Comment by JasonCrowe** [comment]
Here is the machine info:
Pip package: v0.3.8
Camoufox: v132.0-beta.15 (Up to date!)

**Comment by tommylge** [comment]
same here

**Comment by Nongzhsh** [comment]
Playwright isn't thread safe so you need to start playwright separately for each thread.
https://github.com/microsoft/playwright-python/issues/623#issuecomment-840374241

**Comment by JasonCrowe** [comment]
I don't think that is the issue (or I don't understand). I am not trying to open multiple pages at once.

This is what I am doing:
- Create browser
- Open page
- Close browser
- Process data
Repeat...

The problem is when a proxy doesn't load correctly, it goes off the rails and I get the "It looks like you are using Playwright Sync API inside the asyncio loop." error over and over.

Is there a specific pattern I should be following to handle proxy errors?

**Comment by jezonek** [comment]
I have the same issue. It happens sometimes if I try to open/close browser multiple times during processing some data.

**Comment by ev-turenko** [comment]
You need to use the corresponding context for this purpose. Use async Camoufox:

Replace this:
```python
from camoufox import Camoufox
```
by this:
```python
from camoufox.async_api import AsyncCamoufox
```

and follow the examples of usage from the Playwright docs.

**Comment by ev-turenko** [comment]
> You need to use the corresponding context for this purpose. Use async Camoufox:
> 
> Replace this:
> 
> from camoufox import Camoufox
> by this:
> 
> from camoufox.async_api import AsyncCamoufox
> and follow the examples of usage from the Playwright docs.

It works on macos, but it doesn't work on ubuntu...

**Comment by hncjygd** [comment]
Copy MaoOS `$HOME/Library/Caches/camoufox/Camoufox.app/Contents/Resources/addons/UBO` to Linux `$HOME/.cache/camoufox/addons/UBO`。

 tips:  https://github.com/daijro/camoufox/issues/308
